Makefile

```diff
@@ -22,12 +23,18 @@ all:: node_modules package-lock.json
 test:
 	$(LERNA) exec --stream -- $(MAKE) test
 
+test-headless:
+	$(LERNA) exec --stream -- $(MAKE) test-headless
+
 ci:
 	$(LERNA) exec --stream --concurrency=1 -- $(MAKE) test-ci
 
 lint:
 	$(NPX) standard
 
+docker-build:
```

Comment:
I'd be inclined to add a `docker-test-headless` target too that rebuilds and runs the tests in headless docker image